src/main/bash/sdkman-init.sh

```diff
@@ -17,10 +17,12 @@
 #
 
 SDKMAN_PLATFORM="$(uname)"
-if [[ "$SDKMAN_PLATFORM" == 'Linux' && "$(uname -m)" == 'i686' ]]; then
-    SDKMAN_PLATFORM="${SDKMAN_PLATFORM}32"
-elif [[ "$SDKMAN_PLATFORM" == 'Linux' ]]; then
-    SDKMAN_PLATFORM="${SDKMAN_PLATFORM}64"
+if [[ "$SDKMAN_PLATFORM" == 'Linux' ]]; then
+    if [[ "$(uname -m)" == 'i686' ]]; then
+        SDKMAN_PLATFORM+='32'
+    else
+        SDKMAN_PLATFORM+='64'
+    fi
```

Comment:
Does using nested if statements improve performance, or is this merely a stylistic change?

Reply:
It was mainly to avoid repeating the `"$SDKMAN_PLATFORM" == 'Linux'` check.